scripts/globals/mobskills/Slaverous_Gale.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Slaverous Gale
+--
+--  Description: Deals earth damage that inflicts Plague and Slow effects on targets in front of the caster
+--  Type: Magical (Earth)
+--
+--
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target,mob,skill)
+	local typeEffectOne = EFFECT_PLAGUE;
+	local typeEffectTwo = EFFECT_SLOW;
+	MobStatusEffectMove(mob,target,typeEffectOne,1,3,60);
+	MobStatusEffectMove(mob,target,typeEffectTwo,1,3,60);
+	local dmgmod = 1; 
+    local info = MobMagicalMove(mob, target, skill, mob:getWeaponDmg() * math.random(4,6),ELE_EARTH,dmgmod,TP_NO_EFFECT);
```

Comment:
Err, the other one, Gastric bomb.  I'm gonna just go to bed and get up in the morning and make fixes.  Sorry to give you so much work!